cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -2,7 +2,7 @@
 
 import logging
```

Comment:
This file needs to be formatted with Black. I have not setup the pre-commit hook thingy yet either, I think it would make life easier for both of us 😅

Reply:
Yeah, seems as if the auto formatting missed a line break that the GitHub version found necessary. Fixed now.